### DISCUSSION TITLE
Ikea Tradfri E1812 Button Slow to Wake?

### DISCUSSION THREAD

**Discussion by scoobydrvr** [Koenkk/zigbee2mqtt#16545] [Devices]
I’m really not sure where to start with this but it seems others are having this problem in other setups as well… I have 4 of these shortcut buttons around the house and, when they’re working, they work flawlessly, but it seems after some number of hours they stop (as though in a sleep mode) and require several presses (as many as 4) before they start to work again. I’ve pulled up the logs in Z2M and observed that, in this state, they are not reporting the button press to the network. Once they do show something in the logs, the automation fires and everything is back to normal.
Does anyone have a suggestion as to how I can improve this behavior? Is there a setting I can change that will change this, perhaps at the expense of battery life?

**Comment by patrick-87** [comment]
Have you already found a solution by chance? I have this problem too and it's extremely annoying. The on/off switch can also be replaced if necessary, but I like it because of the dimming and I haven't yet found a good alternative for the 5-button remote control 1810 otherwise I would have replaced everything.

**Comment by scoobydrvr** [comment]
No, I never did find a solution to this. Interestingly, though, the issue just kinda went away. All I can offer are my current firmware levels and such:
   Mosquitto: 6.4.0
   Zigbee2MQTT: 1.35.1-1
   Tradfri Button: 24.4.6 build date 20230308

The Release Notes for that version of FW might explain the fix:
   "TRÅDFRI shortcut button (24.4.6)
   Product ID: E1812
   ◆ Optimised battery performance.
   ◆ Stability improvements. "
